wordpress-theme/skyyrose-flagship/data/dossiers/the-fannie.md

```
@@ -20,17 +20,17 @@ extra_references:
 > - Product mockup:                     `assets/products/references/lh-005-fannie-mockup.jpeg`
 
 ### Front
-- **front-body** (the front face of the bag, ~6–8in wide): Solid **black pebbled faux-leather** fabric. **Technique:** stitched. **Color:** black.
-- **front-pocket-zipper** (horizontal, across the front face below the bag's top edge): A black metal/plastic zipper running horizontally across the front face providing access to the front pocket. **Technique:** patch (sewn-on zipper hardware). **Color:** black zipper teeth and pull.
+- **front-body** (the front face of the bag, ~6–8in wide): Solid **black pebbled faux-leather** fabric. **Technique:** stitched. **Color:** black. **Hex:** #000000.
+- **front-pocket-zipper** (horizontal, across the front face below the bag's top edge): A black metal/plastic zipper running horizontally across the front face providing access to the front pocket. **Technique:** patch (sewn-on zipper hardware). **Color:** black zipper teeth and pull. **Hex:** #000000.
 - **front-center-bottom** (~3–4in wide, centered on the lower front face below the zipper): The wordmark **"FANNIE"** in stylized white cursive script lettering, with a small **canonical Love Hurts red rose graphic** decoration to the immediate right of the wordmark (a single small red rose accent — similar in style to the small standalone rose accent used on lh-002 joggers). **Technique:** embroidered (wordmark in white thread + small red rose embroidered detail). **Color:** white "FANNIE" + small red rose accent.
 
 ### Back
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)